wordpress-mobile/WordPress-iOS
# Update GB to version 1.0.0

This PR prepares WordPress to be updated to version 1.0.0 of GB mobile.

To test:
 - All the features in the GB editor
 - Check if the classic editor is still working ok with the new version of Aztec.

Update release notes:
